wire-ios/Wire-iOS Tests/ConversationList/Container/MockConversationListContainer.swift

```diff
@@ -83,6 +83,10 @@ final class MockConversationListContainer: UIViewController, ConversationListCon
         // no-op
     }
 
+    func conversationListViewControllerViewModel(_ viewModel: ConversationListViewController.ViewModel, didUpdate accountImage: (image: UIImage, isTeamAccount: Bool)) {
```

Comment:
Can we format the method into multiple lines?

Reply:
I think it is:
```
tabBarAppearance.shadowImage = .from(solidColor: ColorTheme.Strokes.outline)
```
https://github.com/wireapp/wire-ios/pull/1632/files#diff-80c9932d5dadda7bf1cf0de8548c8f06857c561d3abf90e4ad886474b84493e5R38
<img width="970" alt="Screenshot 2024-07-02 at 08 43 55" src="https://github.com/wireapp/wire-ios/assets/15628584/1a543997-feed-4a26-a7c2-a0b49d976a2b">